src/components/TrancheLine.tsx

```diff
@@ -1,22 +1,22 @@
-import { Fraction } from "@credix/credix-client";
+import { TrancheSettings } from "@consts";
 import { ratioFormatter } from "@utils/format.utils";
+import Big from "big.js";
 import React, { FunctionComponent } from "react";
 
 interface TrancheLineProps {
 	name: string;
 	highlightedElement: string;
 	color: string;
-	apr: Fraction;
-	value: number;
+	structure: TrancheSettings;
```

Comment:
rename. these are settings and we also have a tranchestructure type so this is confusing